x-pack/plugins/apm/server/lib/apm_telemetry/schema.ts

```diff
@@ -187,6 +187,9 @@ export const apmSchema: MakeSchemaFrom<APMUsage> = {
       },
     },
   },
+  service_groups: {
+    kuery_fields: { type: 'array', items: { type: 'keyword' } },
```

Comment:
@dgieselaar Do you foresee any problems in analyising data this way (array of keywords).

Reply:
@dgieselaar I didn't [remove the duplicates](https://github.com/elastic/kibana/pull/132789/files#diff-2a37f786e87b34bd7872b15f82304a2ae2f4792959fe24f757cb8b26943f011dR59).